iothub/device/src/Authentication/AuthenticationWithTokenRefresh.cs

```diff
@@ -101,6 +101,14 @@ public async Task<string> GetTokenAsync(string iotHub)
         {
             if (_isDisposed)
             {
+                if (Logging.IsEnabled)
+                {
+                    Logging.Error(
+                        this,
+                        $"Encountered {nameof(ObjectDisposedException)} - The authentication method instance has already been disposed, so this client is no longer usable.",
```

Comment:
Is this redundant since we are throwing an ObjectDisposedException right afterwards?